acceptance-tests/src/test/java/uk/gov/di/test/pages/RegistrationPage.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.di.test.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import uk.gov.di.test.utils.SignIn;
+
+public class RegistrationPage extends SignIn {
+
+    By radioTextMessageSecurityCodes = By.id("mfaOptions");
+    By secretKeyField = By.id("secret-key");
+    By goBackButton = By.cssSelector("#form-tracking > button");
+    By radioShareInfoAccept = By.id("share-info-accepted");
```

Comment:
I think the id values for accept/reject are different, the got changed a while back and are as follows in the existing step definition: 

Accept:
id("consentValue")
Reject:
id("consentValue-2")